sensitivity-api/src/test/java/com/powsybl/sensitivity/converter/CsvSensitivityComputationResultExporterTest.java

```diff
@@ -0,0 +1,46 @@
+package com.powsybl.sensitivity.converter;
+
+import com.powsybl.commons.AbstractConverterTest;
+import com.powsybl.sensitivity.SensitivityComputationResults;
+import com.powsybl.sensitivity.SensitivityFactor;
+import com.powsybl.sensitivity.SensitivityValue;
+import com.powsybl.sensitivity.json.SensitivityFactorsJsonSerializer;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CsvSensitivityComputationResultExporterTest extends AbstractConverterTest {
+
+    private static SensitivityComputationResults createSensitivityResult() {
+        // read sensitivity factors
+        List<SensitivityFactor> factors = Collections.emptyList();
+        try {
+            factors = SensitivityFactorsJsonSerializer.read(new InputStreamReader(SensitivityComputationResultExportersTest.class.getResourceAsStream("/sensitivityFactorsExample.json")));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
You should avoid to log on std/err, use a logger instead.

Reply:
in that case you need to rethrow the exception stop the test
throw new UncheckedIoException(e)

Reply:
I'll make the change, thanks!